app/packs/src/components/reactQuill/ToolbarDropdown.js

```diff
@@ -50,7 +50,7 @@ export default class ToolbarDropdown extends React.Component {
     });
   }
 
-  UNSAFE_componentWillMount() {
+  componentWillUnmount() {
     try {
       this.sheet.deleteRule(this.ruleId, this.sheet.cssRules.length - 1);
```

Comment:
`deleteRule` does not take a second argument:
https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/deleteRule

Reply:
I have found an other error at the deleteRule and fixed it now